app/content/carto-python/installation.md

```diff
@@ -0,0 +1,58 @@
+---
+aliases:
+    - /carto-python/installation/
+---
+
+## Installation
```

Comment:
I would add links to pydeck-carto and carto-auth:
- https://pydeck-carto.readthedocs.io/en/latest/
- https://github.com/cartodb/carto-auth